Formula/a/afl++.rb

```diff
@@ -0,0 +1,42 @@
+class Aflxx < Formula
+  desc "American Fuzzy Lop++"
+  homepage "https://aflplus.plus"
+  url "https://github.com/AFLplusplus/AFLplusplus/archive/refs/tags/v4.21c.tar.gz"
+  sha256 "11f7c77d37cff6e7f65ac7cc55bab7901e0c6208e845a38764394d04ed567b30"
+  license "Apache-2.0"
+
+  depends_on "coreutils" => :build
+  depends_on "llvm"
+  depends_on "python@3.12"
+
+  def install
+    ENV.prepend_path "PATH", Formula["coreutils"].libexec/"gnubin"
+
+    inreplace "GNUmakefile.llvm", "-flat_namespace", ""
```

Comment:
Let's fix this properly:
```suggestion
    inreplace "GNUmakefile.llvm" do |s|
      s.gsub! "-Wl,-flat_namespace", ""
      s.gsub! "-undefined,suppress", "-undefined,dynamic_lookup"
    end
```